test/core/renderControllerTests.ts

```diff
@@ -1,8 +1,80 @@
 ///<reference path="../testReference.ts" />
 
 describe("RenderController", () => {
-  // HACKHACK: #2083
-  it.skip("Components whose render() is triggered by another Component's render() will be drawn", () => {
+  let SVG_WIDTH = 400;
+  let SVG_HEIGHT = 300;
+
+  describe("configuring the render policy", () => {
+    let oldWarn: (msg: string) => void;
+    let storedWarningMsg: string;
+
+    before(() => {
+      oldWarn = Plottable.Utils.Window.warn;
+      Plottable.Utils.Window.warn = (warningMsg: string) => {
```

Comment:
This is only used in one test, but I see you're using the `before()`/`after()` block. While the `after()` block is good because you won't have a case where `warn` doesn't get restored (due to the test failing), it's odd because other tests in the same `describe()` are affected by this. @aicioara, do you have an opinion on this?